inc/saiswitch.h

```diff
@@ -2776,6 +2776,14 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_ECMP_MEMBER_COUNT,
 
+    /**
+     * @brief Post Ingress ACL stage.
+     *
+     * @type sai_acl_capability_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_ACL_STAGE_POST_INGRESS,
```

Comment:
Why query only for post ingress. HW can be queried for any supported stage for ACL, right?

Reply:
> Why query only for post ingress. HW can be queried for any supported stage for ACL, right?

The attribute exists for other stages - https://github.com/opencomputeproject/SAI/blob/master/inc/saiswitch.h#L1872